cmd/handoff-bss-metadata.go

```diff
@@ -371,19 +371,21 @@ func buildPITArgs(base string) string {
 			cmdlineParts[i] = fmt.Sprintf("hostname=ncn-m001")
 		} else if strings.HasPrefix(part, "ifname=mgmt0") && len(macs) >= 2 {
 			cmdlineParts[i] = fmt.Sprintf("ifname=mgmt0:%s", macs[0])
-		} else if strings.HasPrefix(part, "ifname=mgmt1") && len(macs) >= 2 {
+		} else if strings.HasPrefix(part, "ifname=mgmt1") && len(macs) < 4 {
 			cmdlineParts[i] = fmt.Sprintf("ifname=mgmt1:%s", macs[1])
+                } else if strings.HasPrefix(part, "ifname=mgmt1") && len(macs) >= 4 {
+                        cmdlineParts[i] = fmt.Sprintf("ifname=sun0:%s", macs[1])
```

Comment:
Line 377 should be 
```
cmdlineParts[i] = fmt.Sprintf("ifname=mgmt1:%s", macs[1])
```
We are expecting m002 to be the same hardware as m001.  So if it is HPE, then it will have interfaces mgmt0, mgmt1, sun0, and sun1.